docker/release/build_algod_docker.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+ALGOD_INSTALL_TAR_FILE=$1
+if [[ $ALGOD_INSTALL_TAR_FILE == "" ]]
+then
+   echo "specify filepath of base install file"
+   exit 1
+fi
+
+echo "building 'Dockerfile_algod_release' with install file $ALGOD_INSTALL_TAR_FILE"
+cp $ALGOD_INSTALL_TAR_FILE ./temp_install.tar.gz
+docker build --build-arg ALGOD_INSTALL_TAR_FILE=temp_install.tar.gz . -t algorand/release -f Dockerfile_algod_release
```

Comment:
We also want to tag this so the channel and version are clear.  I'm not sure what the conventions are but we should follow existing conventions.  I would like to be able to provide images for stable, nightly, beta and we want to version them and also have a 'latest' for each channel.

Reply:
Image name now includes the channel and full version with tag 'latest'.